<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsVectorLayerSaveStyleDialog</class>
 <widget class="QDialog" name="QgsVectorLayerSaveStyleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Layer Style</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="mModeLabel">
     <property name="text">
      <string>Save style</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="mStyleTypeComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mStylesWidgetLabel">
     <property name="text">
      <string>Styles</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="mStylesWidget"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mFileLabel">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QgsFileWidget" name="mFileWidget"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="mSaveToDbWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="1">
       <widget class="QgsFileWidget" name="mDbStyleUIFileWidget">
        <property name="toolTip">
         <string>Optionally pick an input form for attribute editing (QT Designer UI format), it will be stored in the database</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="mDbStyleUseAsDefault">
        <property name="text">
         <string>Use as default style for this layer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Style name</string>
        </property>
        <property name="buddy">
         <cstring>mDbStyleNameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="mDbStyleDescriptionEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mUILabel">
        <property name="text">
         <string>UI</string>
        </property>
        <property name="buddy">
         <cstring>mDbStyleNameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mDbStyleNameEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Categories</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QListView" name="mStyleCategoriesListView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mStyleTypeComboBox</tabstop>
  <tabstop>mStylesWidget</tabstop>
  <tabstop>mDbStyleNameEdit</tabstop>
  <tabstop>mDbStyleDescriptionEdit</tabstop>
  <tabstop>mDbStyleUseAsDefault</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsVectorLayerSaveStyleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsVectorLayerSaveStyleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
